myhpom/templates/myhpom/dashboard.html

```diff
@@ -54,7 +54,7 @@ <h2 class="h6 dashboard__section-title">
                     State resident: <strong>{{ request.user.userdetails.state.title|default:'(Unknown state)' }}</strong>
                 </p>
 
-                <div class="dashboard-profile-information__organ-donor-status-box">
+                <div class="dashboard-profile-information__organ-donor-status-box{% if not request.user.userdetails.is_organ_donor %} dashboard-profile-information__organ-donor-status-box--non-donor{% endif %}">
```

Comment:
Not imagination, he definitely indicated the Edit should be removed (not that it can't still be edited, but there should be no encouragement to do so when already a Donor).